less/mixins.less

```diff
@@ -1 +1,23 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.flex (@justify, @align, @flex-direction, @font-size) {
```

Comment:
mixins look great! don't forget that you can set default values to your arguments so you don't need to pass them every time, for example you could write @justify: center, @flex-direction: column, @font-size: 16px on line 2 so when you call .flex you wouldn't need to pass any values and could just write .flex(). You're also still able to pass in new values like you would if you hadn't set defaults.